sql-parser/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/value/literal/impl/StringLiteralValue.java

```diff
@@ -31,4 +32,23 @@ public final class StringLiteralValue implements LiteralValue<String> {
     public StringLiteralValue(final String value) {
         this.value = value.substring(1, value.length() - 1);
     }
+    
+    /**
+     * get special escape sequences string, do this:
```

Comment:
The first letter should be uppercase.

Also, I think we should not keep so many comments, since that's not a feature we provide, it's the ability of `StringUtils.replaceStandardEscapes`.

Reply:
ok

Reply:
Thank you!